.gitignore

```diff
@@ -2,3 +2,4 @@
 *.sh
 .github/kolmafia.jar
 .vs*
+*.swp
```

Comment:
What are swp files? Google says it's created by vim?

Reply:
That they are.  I use gvim for my editing on my Fedora laptop, at least until I get some more memory to double it to 8 GB.  VScode is nice, but it brings my laptop to its knees.